doc/Programming-guide/Key-platform-features/Endpoint-profiles/index.md

```diff
@@ -12,31 +12,27 @@ sort_idx: 20
 
 [Endpoint profile]({{root_url}}Glossary/#endpoint-profile-client-side-server-side) (EP profile) is a structured set of data that describes specific characteristics of an [endpoint]({{root_url}}Glossary/#endpoint-ep).
 
-The structure of both client-side and server-side endpoint profile is a customizable structured data set that describes specific characteristics of the endpoint.
 The profiles are used to attribute endpoints to [endpoint groups]({{root_url}}Glossary/#endpoint-group).
 Every endpoint profile comprises the *client-side*, *server-side* and *system* parts.
 
-The structure of both client-side and server-side parts of an endpoint profile are defined by [application]({{root_url}}Glossary/#kaa-application) developer using the [Apache Avro schema](http://avro.apache.org/docs/current/spec.html#schemas) format.
-Application developer can reuse and share certain data structures using the [common type library (CTL)]({{root_url}}Glossary/#common-type-library-ctl).
+To select an endpoint profile schema, [application]({{root_url}}Glossary/#kaa-application) developer uses the [common type library (CTL)]({{root_url}}Glossary/#common-type-library-ctl).
 
-The client-side structure is used during the SDK generation.
-Therefore, if you make changes to the client-side structure, you need to re-generate your SDK.
+The client-side profile schema is used during the SDK generation.
+Therefore, if you make changes to the client-side schema, you need to re-generate your SDK for the changes to take effect.
 For more information, see [Endpoint SDK]({{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs).
 
 The structure of the system part is identical across the applications and is used by [Kaa platform]({{root_url}}Glossary/#kaa-platform) to perform internal functions.
-Both client-side and server-side profile schemas are maintained within the corresponding application, with its own version that distinguishes it from the previous schemas.
-Multiple schema versions and the corresponding endpoint profiles can coexist within a single application.
 
 ## Client-side EP profile
 
-Initial values for the client-side part are specified by the the client developer using data schemas for the endpoint SDK.
-Then, the client-side endpoint profile is generated during registration of a new endpoint.
+Client developer uses CTL to select a schema for the client-side EP profile.
```

Comment:
Meaning lost here. Previously, the phrase meant that the default contents of the client-side EP profile was defined by the schema itself.